pychromecast/dial.py

```diff
@@ -44,6 +49,13 @@ def get_device_status(host):
     """
 
     try:
+        if not host:
+            for service in services.copy():
```

Comment:
This works. I usually prefer `list(services)` if I'm afraid the source will be mutated during iteration.